### PR TITLE
Convert segsAllConus shapefile to Geojson; create tiles; join csv

### PR DESCRIPTION
**Pull request by mhines-usgs**
for https://github.com/usgs-makerspace/makerspace-sandbox/issues/462

also https://github.com/usgs-makerspace/makerspace-sandbox/issues/463
to create tiles from the geojson

and join with data (temporary data for now) https://github.com/usgs-makerspace/makerspace-sandbox/issues/464

### CODE REVIEW

**Comment by mhines-usgs** [comment]
Jenkins job that uses the Jenkinsfile to run the R script and ogr2ogr transformation https://jenkins.wma.chs.usgs.gov/job/MakerSpace/job/waterUse_streams_convert_shp_to_geojson/

**Comment by mhines-usgs** [comment]
I tested this on my own fork already and you can see the output geojson in s3 here s3://prod-owi-resources/resources/Application/wbeep/test/segsAllConus/segsAllConus.geojson

**Comment by mhines-usgs** [comment]
Added create tiles jenkinsfile - naked tiles go to s3://prod-owi-resources/resources/Application/wbeep/test/segsAllConus/tilesSegsAllConus/ (or to the tier specified in the build job) 
Jenkins job to use the Jenkinsfile https://jenkins.wma.chs.usgs.gov/job/MakerSpace/view/Water%20Temp/job/tippecanoe_build_segsAllConus_tileset/

**Comment by mhines-usgs** [comment]
I put all the jenkins jobs under a new view in jenkins ['water temp'](https://jenkins.wma.chs.usgs.gov/job/MakerSpace/view/Water%20Temp/) 
image

**Comment by mhines-usgs** [comment]
just realized the temp data are not showing up in the join, i think the order of the columns/fields in the csv has to be that the first column is the id we are joining to the tiles, so i'll try manually adjusting that and re-testing before asking Lindsay to modify her script

**Comment by mhines-usgs** [comment]
that helped. will ask Linsday if she can adjust 
image

File: jenkins/tippecanoe_tile_join_segsAllConus_Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        sh 'env'
+        checkout([$class: 'GitSCM',
+                          branches: [[name: "${env.GIT_BRANCH}"]],
+                          doGenerateSubmoduleConfigurations: false,
+                          extensions: [],
+                          gitTool: 'Default',
+                          submoduleCfg: [],
+                          userRemoteConfigs: [[url: 'https://github.com/usgs-makerspace/wbeep-processing']]
+                        ])
+        sh "mkdir ${WORKSPACE}/tilesSegsAllConus"
```
Comment (wdwatkins) [inline jenkins/tippecanoe_tile_join_segsAllConus_Jenkinsfile:30]: could make this one `sh` block with the triple quotes like the other files

**Pull request merged**